docs: update bootstrap doc to reference new CLI commands (#119)
canonical/jaas-documentation: howto/manage-juju-controllers.md

### Prelude

> There are 2 ways to add controllers to JAAS/JIMM:

Reviewer: ```suggestion
Adding a controller to JIMM consists of two steps: bootstrap the controller with the `login-token-refresh-url` controller configuration option, then register it with JIMM. There are two ways you can do that: using `juju` and `juju jaas`, or using `jaas` alone.
```

Author: Going to skip this one and leave it for the refactoring that you mentioned.

> 1. Use the traditional `juju bootstrap` command and then register the controller with JAAS.

Reviewer: ```suggestion
1. Use the traditional `juju bootstrap` command with the `login-token-refresh-url` controller configuration option, then register the controller with JAAS. Note: The `login-token-refresh-url` config option must be set to the specific URL path that serves JIMM's public key, which is used to verify signed requests when they reach the Juju controller.
```

2. Use the `jaas` CLI tool to have JIMM bootstrap a controller without any client-side logic,

> allowing for greater automation. 

Reviewer: In what way is it without any client-side logic? `jaas` is client-side, no? Do you mean that it's JIMM itself talking to clouds and Charmhub to bring the controller to life rather than the client? If so, then this should be made clearer.

Author: Yes, I'll make it clearer.

> In order for a Juju controller to trust a JIMM controller, the `login-token-refresh-url` config option must be set to a specific URL path that serves JIMM's public key, which is used to verify signed requests when they reach the Juju controller.

Reviewer: This should be clarified upstream: https://documentation.ubuntu.com/juju/3.6/reference/configuration/list-of-controller-configuration-keys/#login-token-refresh-url I'd just link there.

Author: True, but for now I don't mind having it here as well since it's highly relevant to JAAS and it deserves a more relevant spotlight then a line in a sea of config options.

> This can be specified manually when bootstrapping the Juju controller directly or is set automatically when bootstrapping through JIMM.

Reviewer: Unnecessary, imo. I'd just make it clear by giving the actual generic workflow under each of 1 and 2.

Author: I'll leave this one as well for the refactoring.

juju jaas register-controller "${CONTROLLER_NAME}" --local --tls-hostname juju-apiserver
```

````

> ````{dropdown} JIMM Bootstrap

Reviewer: ```suggestion
````{dropdown} JIMM bootstrap
```
